sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -12,6 +12,10 @@ public enum ClientSideEncryptionVersion
     {
         V1_0 = 1,
     }
+    public static partial class StorageServerTimeout
+    {
+        public static System.IDisposable CreateScope(int? timeoutInSeconds) { throw null; }
```

Comment:
This should be a `TimeSpan`.

Reply:
I had a timespan here at some point, but [rest doc](https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-blob-service-operations) is calling for integer with pretty small available range.
Isn't `TimeSpan` too broad for this scenario?